go.sum

```diff
@@ -203,6 +203,8 @@ github.com/mtrmac/image/v4 v4.0.0-20191003181245-f4c983e93262/go.mod h1:0ASJH1Yg
 github.com/munnerz/goautoneg v0.0.0-20120707110453-a547fc61f48d/go.mod h1:+n7T8mK8HuQTcFwEeznm/DIxMOiR9yIdICNftLE1DvQ=
 github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223/go.mod h1:qRWi+5nqEBWmkhHvq77mSJWrCKwh8bxhgT7d/eI7P4U=
 github.com/mxk/go-flowrate v0.0.0-20140419014527-cca7078d478f/go.mod h1:ZdcZmHo+o7JKHSa8/e818NopupXU1YMK5fe1lsApnBw=
+github.com/nalind/image/v4 v4.0.0-20191007130334-8f8d9c4a026a h1:mAAtbnQhgmApev4yzxDjv3DsyUO5nAnMFcd3vAOkpqg=
+github.com/nalind/image/v4 v4.0.0-20191007130334-8f8d9c4a026a/go.mod h1:0ASJH1YgJiX/eqFZObqepgsvIA4XjCgpyfwn9pDGafA=
```

Comment:
Are these moving to a non-Nalin location?

Reply:
That's a necessary prerequisite for taking the WIP off of this PR, yes.  That's currently https://github.com/containers/image/pull/400.